lib/x11test.pm

```diff
@@ -974,7 +974,7 @@ sub libreoffice_start_program {
     my %start_program_args;
     $start_program_args{timeout} = 100 if get_var('LIVECD') && check_var('MACHINE', 'uefi-usb');
     x11_start_program($program, %start_program_args);
-    if (match_has_tag('ooffice-tip-of-the-day')) {
```

Comment:
Maybe just add wait_screen_still 5; will fix this unstable issue.

Reply:
check_screen just change on line with wait time. I think it's ok.

Reply:
In fact, match_has_tag and check_screen both work for this part, add wait before catch the needle is the key point for this issue. Just wait_screen_still save the original code design. Anyway, not big concern from my side.